### PR TITLE
Inherit workplace version for sui crates

### PR DESCRIPTION
**Pull request by ebmifa**
## Description 
Inherit the workplace version for sui crates so that the version is the same across all of the binaries

## Test plan 
👀

### CODE REVIEW

**Review by bmwill — CHANGES_REQUESTED** [review]
Lets only do this for the binaries and not for all the libraries (where this isn't really relevant)

**Comment by ebmifa** [comment]
@bmwill Removed all the changes to libraries and left only cli(s)

File: crates/suiop-cli/Cargo.lock [hunk 1]
```diff
@@ -1051,7 +1051,7 @@ checksum = "3e2e65a1a2e43cfcb47a895c4c8b10d1f4a61097f9f254f183aee60cad9c651d"
 
 [[package]]
```
Comment (bmwill) [inline crates/suiop-cli/Cargo.lock:1052]: why is there a lockfile here? either way these changes need to be reverted as you can't change a lockfile like this

Comment (ebmifa) [reply crates/suiop-cli/Cargo.lock:1052]: hmm, i did not touch file. Maybe it got modified by `cargo build --release`

Comment (ebmifa) [reply crates/suiop-cli/Cargo.lock]: Just reverted the changes

**Review by bmwill — APPROVED** [review]

**Pull request merged**